### ISSUE TITLE
reactnative.dev can't be reached

### ISSUE THREAD

**Issue by tkserver** [facebook/react-native-website#2507]
## Description

Website won't load on mac in chrome, firefox, safari, or brave.  Does load on iphone 12. 
<img width="380" alt="Screen Shot 2021-02-16 at 7 58 23 PM" src="https://user-images.githubusercontent.com/3117151/108150826-96a71300-7092-11eb-96be-338a70058b65.png">

This site can’t be reached
reactnative.dev refused to connect.

Try:
Checking the connection
Checking the proxy and the firewall
ERR_CONNECTION_REFUSED

## Documentation version

Current website

**Comment by Simek** [comment]
CC @slorber

**Comment by slorber** [comment]
@tkserver can you please tell me how to reproduce? Have you tried on another computer?

It works for me on all browsers on Mac

**Comment by slorber** [comment]
FYI we did some hosting / DNS changes yesterday, so it may affect you if you have a strict company firewall with ip restrictions or something. Do you think it could be the case? 

Can you give the full details of your environment? 

Is your iPhone on the same connexion as your computer, or are you using another mobile network?

**Comment by tkserver** [comment]
Thanks for the response @slorber 

I had a typo which I fixed, but you can also see in the screen cap. I just accessed the site on my other macbook pro with no issues.  I just tried connecting my MPB 16 to a different network and had the same issue.  I can't hit the site on chrome, brave, safari, or firefox. 

It seems that my 16" MBP has the issue, and I've never been able to access the site for months. Not just recently. Puzzling and frustrating.
<img width="538" alt="Screen Shot 2021-02-17 at 11 31 48 AM" src="https://user-images.githubusercontent.com/3117151/108250875-04475380-7114-11eb-981c-ceb2be986d6a.png">

**Comment by tkserver** [comment]
I figured it out!  Once I realized that this was ONLY my MBP 16" I figured it was a local issue.  So I tried to hit another .dev site like get.dev and it did not work either.  So it wasn't just reactnative.dev. I looked at my hosts file and had this entry below so I could develop locally by going to "mywebsite.dev" instead of typing out the full localhost url:

http://localhost:8888/mywebsite.com/ mywebsite.dev

Once I removed the above line from the hosts file, I was able to hit reactnative.dev.  I'm not a pro on hosts files, but I wouldn't have thought this would affect every .dev site, but it did. 

Thanks for the quick responses.  I'm SO thrilled to be able to access the documentation now. It has been very frustrating looking for answers everywhere but the official docs!
